charts/astronomer/templates/config-syncer/config-syncer-rolebinding.yaml

```diff
@@ -10,6 +10,7 @@
 {{- $useClusterRoles := and .Values.global.rbacEnabled (not .Values.global.features.namespacePools.enabled)}}
 {{- $namespaces := .Values.global.features.namespacePools.namespaces.names }}
 {{- $shouldCreateResources := and .Values.global.rbacEnabled (or (and .Values.global.clusterRoles (not .Values.global.features.namespacePools.enabled )) $namespacepoolsRBACresources) }}
+{{- $data := ( include "configSyncer.ServiceAccount" . )  -}}
```

Comment:
change this to configSyncerServiceAccountName  as variable declaration

Reply:
done